### ISSUE TITLE
party create error

### ISSUE THREAD

**Issue by adraaawr** [OpenKore/openkore#1408]
iRO: Restart
Problem on creating party using kore, 

![dafuq](https://user-images.githubusercontent.com/32107546/32946056-e23c4db8-cbd0-11e7-8557-a5ec6883acf9.png)

party request works when you are already a party leader. Need fix. Thx

**Comment by sctnightcore** [comment]
**Download new openkore and test again ! .**

**Comment by adraaawr** [comment]
@sctnightcore I am using the latest openkore.

**Comment by sctnightcore** [comment]
@adraaawr  Did you edit timeout.txt?

**Comment by alisonrag** [comment]
its not relationed to timeouts, maybe the party is not unset whe you leave

**Comment by adraaawr** [comment]
Nope, its fresh openkore folder @sctnightcore , when you type 'party leave' in kore, nothing happens. I tried using the old commands.pm - and it works.

**Comment by sctnightcore** [comment]
`party [<info|create|join|request|leave|share|shareitem|sharediv|shareauto|kick|leader>]`
I think you should try new.
You can upload old commands.pm?

**Comment by adraaawr** [comment]
I tried the new one, I fetched openkore multiple times, and `party create <partyname>` dont work, i need to manually create a party ingame to invite my bots using kore.

But the old commands.pm `party create` works.


[Commands.zip](https://github.com/OpenKore/openkore/files/1482515/Commands.zip)

**Comment by fadreus** [comment]
I tested PR from #1383 .
When leaving party, it actually leave but kore didn't recognize it.
Relogging solved this issue.

Same ID, different src/Commands.pm
![image](https://user-images.githubusercontent.com/32188494/32951674-c26474f6-cbe5-11e7-8a5c-db1d8167593f.png)

After relogging
![image](https://user-images.githubusercontent.com/32188494/32951697-db959194-cbe5-11e7-9b61-7470800d5f59.png)

**Comment by lututui** [comment]
I can't reproduce it

This test was made in bRO
![image](https://user-images.githubusercontent.com/17522256/32951816-6f31abe4-cb92-11e7-969d-6dbc6e3e40ae.png)

**Comment by fadreus** [comment]
@lututui Guess it doesn't work for all servers? :thinking: :thinking:
@adraaawr You can revert to previous version for file src/Commands.pm if you need fast solution.

**Comment by adraaawr** [comment]
@fadreus , thanks. Im currently using the previous version of the Commands.pm, will wait for further updates. thanks!

**Comment by lututui** [comment]
Even though the bug report template asks you to post this, you clearly ignored it: what server it this?

**Comment by adraaawr** [comment]
@lututui , sorry. iRO restart.

**Comment by sctnightcore** [comment]
**Please use the issue template, please post infornations about the server and your configuration.**

**Comment by lututui** [comment]
This problem is not about #1383, but rather why $char->{party} is not getting deleted when we leave the party in iRO Restart

**Comment by sctnightcore** [comment]
**openkore/src/Network/Send/iRO.pm** @lututui  
```
	my %handlers = qw(
		sync 0360
		character_move 035F
		actor_info_request 0368
		actor_look_at 0361
		item_take 0362
		item_drop 0363
		storage_item_add 0364
		storage_item_remove 0365
		skill_use_location 0366
		party_setting 07D7 << ? 
		buy_bulk_vender 0801
		char_delete2_accept 098f
		send_equip 0998
	);
	$self->{packet_lut}{$_} = $handlers{$_} for keys %handlers;

	$self->{char_create_version} = 0x0A39;
	
	return $self;
}
```

**Comment by lututui** [comment]
@fadreus could you repeat the test you made in the top right image, but replace **sub party_leave** in src/Network/Receive/ServerType0.pm with this one https://gist.github.com/lututui/0706cfc547654f92daec4918277db7c8

@sctnightcore it's not a packet problem, we're actually leaving the party

**Comment by fadreus** [comment]
> https://gist.github.com/lututui/0706cfc547654f92daec4918277db7c8

@lututui Seems fine with that :thinking: :thinking: :thinking: ...
Ignore that `!�R` 🤣

![image](https://user-images.githubusercontent.com/32188494/32954037-e549db58-cbec-11e7-9124-4d23af69cd22.png)

**Comment by lututui** [comment]
@fadreus I did nothing to it though, I just added the message printing to see if both values were equal

**Comment by fadreus** [comment]
@lututui It only appear when leaving party. My Perl is v5.24.1 but I use start.exe.

**Comment by lututui** [comment]
@fadreus @adraaawr I think #1409 fixes this

**Comment by fadreus** [comment]
Fixed with #1409 👌👌👌
☺️☺️☺️

**Comment by alisonrag** [comment]
Fixed
